src/gcp/firedata.ts

```diff
@@ -20,29 +20,7 @@ function _handleErrorResponse(response: any): any {
 }
 
 /**
- * Create a new Realtime Database instance
- * @param projectId Project from which you want to get the ruleset.
- * @param instanceName The name for the new Realtime Database instance.
- */
-export async function createDatabaseInstance(
-  projectNumber: number,
-  instanceName: string
-): Promise<any> {
-  const response = await api.request("POST", `/v1/projects/${projectNumber}/databases`, {
-    auth: true,
-    origin: api.firedataOrigin,
-    json: {
-      instance: instanceName,
-    },
-  });
-  if (response.status === 200) {
-    return response.body.instance;
-  }
-  return _handleErrorResponse(response);
-}
-
-/**
- * Create a new Realtime Database instance
+ * List Realtime Database instances
  * @param projectId Project from which you want to get the ruleset.
  * @param instanceName The name for the new Realtime Database instance.
```

Comment:
These parameters aren't what listDatabaseInstances takes though. It should look like

```suggestion
 * @param projectNumber Project from which you want to list databases.
 * @return the list of databases.
```

Reply:
Thanks - sorry I didn't catch that.